templates/app-ts/src/app.ts

```diff
@@ -1,11 +1,8 @@
 import { join } from 'path';
 import AutoLoad, {AutoloadPluginOptions} from '@fastify/autoload';
-import { FastifyPluginAsync } from 'fastify';
-
-export type AppOptions = {
-  // Place your custom options for app below here.
-} & Partial<AutoloadPluginOptions>;
+import { FastifyPluginAsync, FastifyServerOptions } from 'fastify';
 
+export type AppOptions = FastifyServerOptions & Partial<AutoloadPluginOptions>;
```

Comment:
It is not extendable as the previous one. Using `interface ... extends` would be better.

```ts
interface AppOptions extends FastifyServerOptions, Partial<AutoloadPluginOptions> {

}
```

Reply:
@climba03003 thanks, I have updated the code.